src/test/scala/com/cognite/sdk/scala/v1/FilesTest.scala

```diff
@@ -24,6 +24,8 @@ class FilesTest extends SdkTestSpec with ReadBehaviours with WritableBehaviors w
 
   it should behave like readableWithRetrieveByExternalId(client.files, externalIdsThatDoNotExist, supportsMissingAndThrown = true)
 
+  it should behave like readableWithRetrieveUnknownIds(client.dataSets)
```

Comment:
lifted this pattern from the asset tests - not exactly sure why we use the datasets as input

Reply:
I'm not sure either, tbh.